lib/widgets/picos_add_button_bar.dart

```diff
@@ -0,0 +1,73 @@
+/*   This file is part of Picos, a health tracking mobile app
+*    Copyright (C) 2022 Healthcare IT Solutions GmbH
+*
+*    This program is free software: you can redistribute it and/or modify
+*    it under the terms of the GNU General Public License as published by
+*    the Free Software Foundation, either version 3 of the License, or
+*    (at your option) any later version.
+*
+*    This program is distributed in the hope that it will be useful,
+*    but WITHOUT ANY WARRANTY; without even the implied warranty of
+*    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*    GNU General Public License for more details.
+*
+*    You should have received a copy of the GNU General Public License
+*    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import 'package:flutter/material.dart';
+import 'package:picos/widgets/picos_ink_well_button.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import '../themes/global_theme.dart';
+
+/// Class for the Add-Button.
+class PicosAddButtonBar extends StatelessWidget {
```

Comment:
Personal opinion here: I don't see any reason one should put "picos" in front of any class names. Furthermore if you're putting a prefix everywhere it's a clear indicator to get rid of it. I'd suggest to refactor this, but keep in mind that just me being opinionated.

Reply:
> Furthermore if you're putting a prefix everywhere it's a clear indicator to get rid of it.
Yeah, that's right actually 🤔 

I think the prefix 'Picos' denotes that these are classes specially developed for Picos and that we didn't use other libraries.

Reply:
I've started naming classes for general use with prefix, so you can directly differ between our classes and some one else's/framework classes. That is especially useful, if you take an already existing class extending upon it, so you don't have to think for a new class name, just use a prefix. But there are thousands different conventions you could or could not follow regarding naming. The "Picos" bit can just as well be removed.